README.arm.md

```diff
@@ -125,47 +132,32 @@ These tutorials will end up installing Ubuntu 12.04, and you have to
 upgrade to Ubuntu 14.04, or install Ubuntu 14.04 from scratch by
 finding appropriate `crouton` help.
 
-## Scaleway cloud hosted ARM servers
+#### Scaleway cloud hosted ARM servers
 
 On the current [Scaleway](http://scaleway.com) ARM servers, the Julia
 build works out of the box.
 
-## nVidia Jetson TX2
+## AArch64 (ARMv8)
 
-Julia builds and runs on the [nVidia Jetson TX2](http://www.nvidia.com/object/embedded-systems-dev-kits-modules.html) platform with minimal configuration changes. A full multi-threaded build, including LLVM, will complete in around two hours. All tests pass and CUDA functionality is available through, e.g., [CUDAdrv](https://github.com/JuliaGPU/CUDAdrv.jl).
+### Device specific instructions
 
-Starting from the default configuration flashed by [Jetpack 3.0](https://developer.nvidia.com/embedded/jetpack):
+#### nVidia Jetson TX2
 
-```
-sudo apt-get install libssl-dev
-```
+Julia builds and runs on the [nVidia Jetson TX2](http://www.nvidia.com/object/embedded-systems-dev-kits-modules.html)
+platform with minimal configuration changes. A full multi-threaded build, including LLVM,
+will complete in around two hours. All tests pass and CUDA functionality is available
+through, e.g., [CUDAdrv](https://github.com/JuliaGPU/CUDAdrv.jl).
 
-### Julia 0.5.1
-
-The easiest method to build Julia 0.5.1 is to use system provided versions of BLAS and LAPACK:
-
-```
-sudo apt-get install libopenblas-dev liblapack-dev
-```
-
-Configure Make.user as follows:
+Starting from the default configuration flashed by [Jetpack 3.0](https://developer.nvidia.com/embedded/jetpack):
 
 ```
-MARCH=armv8-a
-JULIA_CPU_TARGET=cortex-a57
-override USE_SYSTEM_BLAS=1
-override USE_SYSTEM_LAPACK=1
+sudo apt-get install libssl-dev
```

Comment:
The question is more about which dependency links to libssl rather than what package to install. If libcurl is the one that needs libssl, it should be mentioned in the main README.